src/theme.js

```diff
@@ -55,6 +55,8 @@ const theme = createTheme({
   palette: {
     background: {
       default: '#FFFFFF',
+      greenBg:
```

Comment:
@VWRoli I was thinking that to reuse/remind all these customized theme settings for future reuse, we need a sandbox to show them all. Like this issue I mentioned: #301 so people can easily pick what they want.

Reply:
Yes, we definitely need to do something with the colors and typography. They are a bit all over the place and illogical sometimes. 
For example, the primary color should be green and not white.
And also in the typography section, the subtitle variant should not be 36px big.
I might take a look at the issue and try to come up with something.